tests/utils/test_file.py

```diff
@@ -212,3 +212,17 @@ def test_get_modules_from_invalid_file(self):
         modules = list(file_utils.iter_airflow_imports(file_path))
 
         assert len(modules) == 0
+
+
+def test_get_unique_dag_module_name():
```

Comment:
nitpick: I think we can check the expected result as well so folks would know what's the expected output

Reply:
resolve: Since there `SHA-1` of `file_path` in the expected output, so I use regex to validate instead